partcad/src/partcad/project_factory_git.py

```
@@ -212,7 +212,9 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
                             )
                             time.sleep(patience)
                         else:
-                            pc_logging.error("Failed to update repo after %s retries" % attempt)
+                            pc_logging.error(
+                                "Failed to update repo %s after %d retries", self.import_config_url, attempt
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)